samples/browser/build.gradle

```diff
@@ -80,7 +80,7 @@ android {
             setIgnore(true)
         }
 
-        if (flavors.contains("servo") && !flavors.contains("arm")) {
+        if (flavors.contains("servo") && (flavors.contains("universal") || flavors.contains("aarch64"))) {
```

Comment:
Let's reverse that check to check for ARM or x86. Otherwise this needs to be updated for every new ABI we add (x86_64 may get added soon).

Reply:
Will do in 8 hours.

Reply:
fixed